### DISCUSSION TITLE
Invalid Command or arguments for dns in OTBR

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#9380] [Q&A] [answered]
Hi all,
I build OTBR in ubuntu, but these commands seem not work, does anyone knows how to support these commands? Thanks

![c7347c258bfd48afb39425ded410161](https://github.com/openthread/openthread/assets/30258768/03e06ef0-5dea-4d66-8a8b-1f5d7e98182e)

**Comment by abtink** [comment]
Your build does not seem to have `OPENTHREAD_CONFIG_DNS_CLIENT_ENABLE` feature enabled.
CMake option for this config -> `OT_DNS_CLIENT`

**Comment by Leo2442926161** [comment]
Thanks Abtink,
Does that mean those commands only work for DNS client? 
In the OTBR, I think it suppose to be DNS server.

**Reply by abtink** [reply, marked as answer]
Yes. The "dns config" and "dns browse" are part of "DNS client" feature.

You can enable both features. A device can act as DNS server and a DNS client.

**Reply by Leo2442926161** [reply]
Thanks very much Abtink.
